src/sqlacodegen/cli.py

```diff
@@ -24,13 +25,40 @@
     pgvector = None
 
 if sys.version_info < (3, 10):
-    from importlib_metadata import entry_points, version
+    from importlib_metadata import EntryPoint, entry_points, version
 else:
-    from importlib.metadata import entry_points, version
+    from importlib.metadata import EntryPoint, entry_points, version
 
 
-def main() -> None:
-    generators = {ep.name: ep for ep in entry_points(group="sqlacodegen.generators")}
+_generators_cache = None
+
+
+def get_generators() -> dict[str, EntryPoint]:
```

Comment:
How is this needed or related to this PR in any way. Please remove this piece of code, on focus on relevant changes only